content/en/tracing/trace_retention_and_ingestion/usage_metrics.md

```diff
@@ -42,5 +42,5 @@ Because Indexed Spans can impact your bill, the 'Spans Indexed' column appears a
 [2]: /tracing/trace_retention_and_ingestion/#ingestion-controls
 [3]: https://www.datadoghq.com/pricing/?product=apm#apm
 [4]: /account_management/billing/apm_distributed_tracing/
-[5]: https://app.datadoghq.com/dash/integration/30337/trace-analytics---usage
+[5]: https://app.datadoghq.com/dashboard/lists?q=APM+Traces+-+Estimated+Usage
```

Comment:
it seems that the query params are not picked up in the link.. is this a known issue ? 🤔

Reply:
Made a Jira issue for us to thoroughly investigate as we don't have an immediate answer. Do you want us to hold on merging until resolved?

Reply:
yup, there is no rush really on this particular link.. it was "more convenient" for non-US-1 DD sites but it can way

Reply:
Added a fix for the link query string params to this branch. If you would prefer that a separate PR @sarina-dd let me know!

Reply:
This is perfect, thanks @nsollecito! 😸